### DISCUSSION TITLE
Voxel Capabilities?

### DISCUSSION THREAD

**Discussion by wattahay** [bevyengine/bevy#2714] [Q&A] [answered]
Is there (or can there be) deeply integrated voxel functionality planned?

Are voxel engines only efficient as their own engines?

There are a ton of voxel engines - not so many good multi-core, efficient, well developed ones with a big community.

**Comment by afonsolage** [comment, marked as answer]
Bevy is not focused on Voxel Engine, but there is a big crate that has bevy integration and some comunnity projects where you can learn from:

The crate: 

- https://github.com/bonsairobo/building-blocks

The projects: 

- https://github.com/superdump/minkraft
- https://github.com/Game4all/vx_bevy

**Reply by wattahay** [reply]
Thank you, I'll check those out.

**Reply by afonsolage** [reply]
If this answer your question, feel free to mark it as answered, so others can check it also :smile:

**Reply by wattahay** [reply]
Sorry, this is my first time doing this. The interface is not letting me do it. 

The button "Mark as answer" is darkening on hover, but nothing happens when I click it. I see a dark grey check mark next to it, but at the top under the title I still see "Unanswered". Additionally, I cannot get rid of bubble message telling to mark it. 


I tried it on Chrome and Firefox. I am very frustrated.

**Reply by bjorn3** [reply]
Weird. In any case, it seems that I was able to mark it for you.
